l5kit/l5kit/data/map_api.py

```diff
@@ -176,6 +272,21 @@ def get_crosswalk_coords(self, element_id: str) -> dict:
 
         return {"xyz": xyz}
 
+    def is_traffic_light(self, element_id: str) -> bool:
```

Comment:
```suggestion
    def has_traffic_light(self, element_id: str) -> bool:
```

Reply:
all other (lanes, crosswalks) functions are `is` and not `has`. I think `is` makes more sense as you're checking what the element is in this case..

Reply:
okay, I misunderstood it.